### DISCUSSION TITLE
建议IC2在未来版本中为各类工具加入<直接性了解工具或机器此时状态>的直观描述功能等建议。

### DISCUSSION THREAD

**Discussion by WaterDouble** [IndustrialCraftMC/IndustrialCraft2#44] [【意见建议】Ideas]
1.众所周知，IC2是个复杂多变的生电模组，由于其复杂性，这时其就显得格外重要。<直接性了解工具或机器此时状态>可以为众多机器或工具提供最基本的信息状态描述，这可以避免很多“需要手动实验是否在此状态”的问题。就比如说“量子套是否处于飞行模式”、"火力发电机内有什么"，通过其我便可以直接明了的知道是否处于飞行模式。
2.可以为量子套和纳米套加入相对应的“画面特效”，使玩家穿戴全套盔甲时能感受到高级铠甲带来的科技炫酷感，从而更加有成就感。

**Comment by AbbottMc** [comment]
套装的动画优化是在后续套装升级计划范围内的，可以期待一下
